packages/web/src/components/Shelves/FlowerItemAdd/FlowerItemAdd.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FlowerItemAdd } from '../FlowerItemAdd';
```

Comment:
Git suggest next changes:
Useless path segments for "../FlowerItemAdd", should be "./"